test/integration/patients/patient/sidebar.js

```diff
@@ -177,6 +178,15 @@ context('patient sidebar', function() {
               emptyField: '@patient.emptyField',
             },
           }),
+          addWidget({
+            slug: 'hbsNoRegionWidget',
+            category: 'widget',
+            definition: {
+              template: 'Content that will not appear because the region is missing',
+              // Missing <div data-content-region> on purpose
+              wrapperTemplate: '<div class="no-region-wrapper">No region defined</div>',
+            },
+          }),
```

Comment:
Do we need to add another widget to the test that's similar to the one you added in the README?

Where the `wrapperTemplate` has logic that shows/hides the widget based on a value. And also verifies that it renders correctly when the `<div data-content-region>` region is included in the `wrapperTemplate`.